### PR TITLE
AOM-106: Externalize Addon URL

### PR DESCRIPTION
**Pull request by Charpell**
### SUMMARY
The ticket requires going through the entire codebase to look for all instances of addon endpoints and replacing them with a constant defined in a helper file.

### CODE REVIEW

**Comment by Farhanisto** [comment]
Code looks good

File: app/js/helpers/apiHelper.js [hunk 1]
```diff
@@ -69,3 +69,5 @@ export class ApiHelper {
     return this.build(requestUrl, 'DELETE').send();
   }
 }
+
+export const BASE_URL = "https://addons.openmrs.org/api/v1/addon";
```
Comment (kodero) [inline app/js/helpers/apiHelper.js]: Refactor looks fine. Did you consider putting the `BASE_URL` as a static member in the `ApiHelper` class? That way, there's only one import variable.

**Comment by malmike** [comment]
Looks Good

File: app/js/helpers/apiHelper.js (hunk 1 above)
Comment (Charpell) [reply app/js/helpers/apiHelper.js]: Thank you for the observation, i have also updated the PR

**Comment by kodero** [comment]
Looks good!

File: app/js/components/manageApps/ManageApps.jsx [hunk 2]
```diff
@@ -158,7 +158,7 @@ export default class ManageApps extends React.Component {
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
         return Promise
-          .all(response.data.results.map(data => axios.get(`https://addons.openmrs.org/api/v1/addon/${data.packageName}`)
```
Comment (dkayiwa) [inline app/js/components/manageApps/ManageApps.jsx:161]: Is addonUrl is property or a method that gets the url?
In other words, if a method, isn't getAddonUrl clearer?

Comment (Charpell) [reply app/js/components/manageApps/ManageApps.jsx:161]: Yes its clearer, i have updated the PR

**Pull request merged**